bndtools.core/src/bndtools/central/Central.java

```diff
@@ -253,6 +258,14 @@ public static Workspace getWorkspace() throws Exception {
 					ws.addBasicPlugin(getEclipseWorkspaceRepository());
 					ws.addBasicPlugin(new JobProgress());
 
+					String tabulationChar = prefsService.getString("org.eclipse.jdt.core.prefs",
+						"org.eclipse.jdt.core.formatter.tabulation.char", "tab", null);
```

Comment:
Is this really the formatting properties to use for POMs?

Reply:
fixed